src/components/AssetInput/AssetInput.spec.tsx

```diff
@@ -0,0 +1,264 @@
+        source: "library",
+        sourceName: "",
+    },
+    {
+        name: "foo2",
+        size: 2000,
+        type: "image",
+        extension: "JPG",
+        src: "https://picsum.photos/200/200",
+        source: "library",
+        sourceName: "",
+    },
+    {
+        name: "foo3",
+        size: 2000,
+        type: "image",
+        extension: "JPG",
+        src: "https://picsum.photos/100/100",
+        source: "library",
+        sourceName: "",
+    },
+    {
+        name: "foo4",
+        size: 1000,
+        type: "image",
+        extension: "JPG",
+        src: "https://picsum.photos/400/200",
+        source: "library",
+        sourceName: "",
+    },
+    {
+        name: "foo5",
+        size: 1000,
+        type: "image",
+        extension: "JPG",
+        src: "https://picsum.photos/400/200",
+        source: "library",
+        sourceName: "",
+    },
+];
+
+const MIXED_ASSETS = EXAMPLE_IMAGES.slice(0, 2);
+
+MIXED_ASSETS.push(
+    {
+        source: "library",
+        sourceName: "Foobar",
+        name: "foo",
+        type: "icon",
+        icon: <IconIcons />,
+    },
+    {
+        source: "library",
+        sourceName: "Foobar",
+        name: "foo",
+        extension: "MP3",
+        size: 2000,
+        type: "audio",
+    },
+);
+
+describe("MultiAssetPreview Component", () => {
+    it("renders asset input placeholders", () => {
+        mount(
+            <AssetInput
+                size={AssetInputSize.Small}
+                onLibraryClick={() => alert("Library clicked")}
+                onUploadClick={() => alert("Upload clicked")}
+            />,
+        );
+
+        cy.get(ASSET_PLACEHOLDER_ID).should("exist");
+        cy.get(ASSET_PLACEHOLDER_UPLOAD_ID).should("exist");
+        cy.get(ASSET_PLACEHOLDER_LIBRARY_ID).should("exist");
+    });
+
+    it("renders asset input with single image", () => {
+        mount(
+            <AssetInput
+                size={AssetInputSize.Small}
+                assets={[
+                    {
+                        source: "upload",
+                        name: "foo",
+                        size: 2000,
+                        type: "image",
+                        extension: "JPG",
+                        src: "https://picsum.photos/200/300",
+                    },
+                ]}
+                actions={actions}
+            />,
+        );
+
+        cy.get(ASSET_SINGLE_INPUT_ID).should("contain", "2000");
+        cy.get(ASSET_SINGLE_INPUT_ID).should("contain", "Upload");
+        cy.get(ASSET_SINGLE_INPUT_ID).should("contain", "JPG");
+        cy.get(ASSET_SINGLE_INPUT_THUMBNAIL_ID).should("have.css", "backgroundColor");
+        cy.get(ASSET_SINGLE_INPUT_THUMBNAIL_IMAGE_ID).should("have.attr", "src", "https://picsum.photos/200/300");
+    });
+
+    it("renders asset input with audio", () => {
+        mount(
+            <AssetInput
+                size={AssetInputSize.Small}
+                assets={[
```

Comment:
I'd reuse one of the existing example images and not create a new one, e.g. `EXAMPLE_IMAGES[0]`